### PR TITLE
complete contact page

### PR DESCRIPTION
**Pull request by Ritu-03Thakur**
changes -:
1. Index.html
2. style.css
3. add photos
contact_page1

### CODE REVIEW

**Comment by Suvradippaul** [comment]
Hey @Ritu-03Thakur, I have created a new branch 'test'. Please give PRs to that branch. I need to check few things before merging to 'main' branch.

File: style.css [hunk 1]
```diff
@@ -407,4 +407,91 @@ footer.footer ul li a{
     border-radius: 9px;
     position: relative;
     left: 1118px;
-}
\ No newline at end of file
+}
+/* Contact US */
```
Comment (Suvradippaul) [inline style.css:411]: Please make it responsive

File: style.css [hunk 2]
```diff
@@ -407,4 +407,91 @@ footer.footer ul li a{
...
+  top : 50px;
+    width: 40%;
+    height: 300px;
+    position: relative;
+    border-radius: 10px;
+    left: 500px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    background-color: rgb(65, 1, 1);
```
Comment (Suvradippaul) [inline style.css:444]: Please maintain the color theme

**Review by Suvradippaul — CHANGES_REQUESTED** [review]

**Comment by Suvradippaul** [comment]
@Ritu-03Thakur Its conflicting

**Comment by Ritu-03Thakur** [comment]
Should I make another pull request to main branch?

**Comment by Suvradippaul** [comment]
> Should I make another pull request to main branch?

Yes

**Pull request closed without merging**